src/plugins/irc-events/topic.js

```diff
@@ -9,11 +9,9 @@ module.exports = function(irc, network) {
 			return;
 		}
 		var from = data.nick || chan.name;
-		var self = false;
-		if (from.toLowerCase() == irc.me.toLowerCase()) {
-			self = true;
-		}
+		var self = (from.toLowerCase() == irc.me.toLowerCase());
```

Comment:
Looks like someone has a style preference here :P

So, since you want to play that game, let me be picky ^^

- **Always** use `===` over `==`. There are effectively no good reasons to use the latter over the former (apart from saving one character or using the side-effects of `==` on purpose, none of which are good reasons) but there are plenty good reasons to use the former over the latter.
- Because of the way JavaScript treats `this`, it is extremely common to see functions starting with `var self = this;`. Therefore, `self` is generally reserved for this purpose (along with `that`, but it is frankly less common). See http://jscs.info/rule/safeContextKeyword for example. Let's keep our codebase future-proof when world-class JS developers jump in with their habits :-)

In conclusion, add to this that `self` is just used once, please move this block to line 22 such that:

```js
  self: from.toLowerCase() === irc.me.toLowerCase()
```

(NB: @JocelynDelalande [DRYed this up](https://github.com/astorije/shout/commit/6165795d7c90d80f9be24ce2f7406234879be398#diff-4b7d8172f3d87ca98fe952995318443aR61), but it will be for later when it makes it to master)